target/linux/ath79/image/generic.mk

```diff
@@ -1813,6 +1813,26 @@ define Device/hiwifi_hc6361
 endef
 TARGET_DEVICES += hiwifi_hc6361
 
+define Device/huawei_ap5030dn
+  SOC := qca9550
+  DEVICE_VENDOR := Huawei
+  DEVICE_MODEL := AP5030DN
+  DEVICE_PACKAGES := ath10k-firmware-qca988x-ct kmod-ath10k-ct
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0x110040
+  IMAGE_SIZE := 7680k
+  ROOTFS_MIN_SIZE := 12240k
```

Comment:
This is weird, image-size < rootfs-min-size?

Reply:
Removed this development leftover.